src/android/ContentFilesystem.java

```diff
@@ -35,6 +35,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 public class ContentFilesystem extends Filesystem {
 
     private final Context context;
+    private final String PHOTO_PICKER_PREFIX = "/synthetic/picker_get_content/";
```

Comment:
I think there may be devices where this URI is a little different. For example, I just tested on a Samsung A53 and the URI is `synthetic/picker/` instead.